vendor/golang.org/x/sys/unix/asm_bsd_riscv64.s

```diff
@@ -0,0 +1,29 @@
+// Copyright 2021 The Go Authors. All rights reserved.
```

Comment:
Did you update vendor/ `go mod` command? If yes, I would expect some go.mod/go.sum changes as well.
If not, then this will be reverted after the next update.

Reply:
Update go.mod in https://github.com/google/syzkaller/pull/3253/commits/c49e64da454e336aaa4db8a69aec06e9cf1fac2c.